### PR TITLE
Aden 3761 wait for page load

### PR DESCRIPTION
**Pull request by jacekjursza**
- extend method AdsBaseObject.waitForPageLoaded to wait for gpt is loaded

- split tests for project43 and adtest-pluto in order to use mockAds=true on adtest-pluto

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java [hunk 1]
```diff
@@ -110,7 +112,28 @@ public void adsAtfOnTabletOasis(String wikiName, String article, boolean isWgVar
                          AdsContent.PREFOOTER_RIGHT);
   }
 
-  @RelatedIssue(issueID = "ADEN-3761")
+  @Execute(mockAds = "true")
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java]: have you considered using:
`
@InBrowser(
    emulator = Emulator.GOOGLE_NEXUS_5,
    browser = Browser.CHROME
)
` ?

**Comment by bognix** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_left.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/c9c7a4c6ed8915dd16ff6a21117b208179a60cb7/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L174) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_left.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/c9c7a4c6ed8915dd16ff6a21117b208179a60cb7/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L174) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)

**Pull request merged**